README.md

```diff
@@ -1,3 +1,6 @@
+# DO NOT USE - Temporary Fork #
```

Comment:
I understand why you put this comment note here, but ideally it would be gone before this PR was merged. Maybe use a separate branch downstream for this PR?

Reply:
Yeah makes sense, I'll reopen with a new branch.